eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyCachingTest.java

```diff
@@ -59,4 +62,26 @@ void putsObjectToLocalCache(@TempDir final Path path) throws Exception {
                 .exists()
         );
     }
+
+    @Test
+    void checksPresenceOfObject(@TempDir final Path path) throws Exception {
+        final String content = "[] > main\n";
+        new LengthOf(
```

Comment:
@MikhailLipanin Why do you need `LenghtOf` here?

Reply:
@volodya-lombrozo, `TeeInput` is just a constructor. We need to "execute" it. It can be done by encapsulating it in `LengthOf` and calling method `value()`. This trick is used for example in `Home` class in method `save()`

Reply:
@MikhailLipanin Why do you need `Input` here then?

Reply:
@volodya-lombrozo The goal of this test is: to save manually some object in the "temp" directory, than in constructor of `OyCaching`, there is a `OyHome`, which is binded to Path, on which we already saved an object manually. Then, encapsulating `OyHome` in `OyCaching`, we call `contains()` method, which checks: does method `contains()` in `OyCaching` works well?

Reply:
in other words, `OyCaching` will "ask" its "primary" objectionary, that is `OyHome` --- does it have the object?

Reply:
@volodya-lombrozo, asking a question, To save manually an object --- we need this construct with `TeeInput`, which just knows where to find text to `read` and where to `write` it. And execute it by `LengthOf().value()`

Reply:
@MikhailLipanin I believe it's an overdesign here:
1. You can save content by `path` using simple `Home.save`
2. You can inline `content` variable
3. Then you can just your `path` to the `OyCaching` as you doing it right now

Reply:
done